tests/e2e/scripts/cloud-run/deploy.sh

```diff
@@ -116,6 +118,19 @@ function deployBackend() {
 
       BACKEND_HOST=$(gcloud functions describe ${BACKEND_SERVICE_NAME}  --format="value(httpsTrigger.url)" --quiet)
       ;;
+    "app-engine")
+    cd ${ROOT}/tests/endpoints/bookstore
+
+    sed "s/SERVICE_NAME/${BACKEND_SERVICE_NAME}/g" app_template.yaml > app.yaml
+    gcloud app deploy --quiet
```

Comment:
Is this app IAP enabled?   If no,  can we enable IAP?

Reply:
Yes, it is using IAP.